### DISCUSSION TITLE
Sort PRs descending

### DISCUSSION THREAD

**Discussion by kristijanhusak** [pwntester/octo.nvim#336] [Q&A] [answered]
Hi,

Is there a way to somehow sort PRs in descending order?

I see in github docs that sorting is supported https://docs.github.com/en/graphql/reference/input-objects#issueorder,  but I don't know if there's a way to pass it through octo. From what I understand it supports only filters.

**Comment by pwntester** [comment]
Hi, thanks for the suggestion, that sounds feasible. Do you mind opening an issue for it?

**Comment by williambdean** [comment, marked as answer]
This is possible via configuration. This configuration is for issues, pull requests, and discussions. Find it here: https://github.com/pwntester/octo.nvim/blob/90304a39703c545aa481d68643ed34dad5379a8b/README.md?plain=1#L137-L142
